pkg/js/helpers.js

```diff
@@ -1850,7 +1850,7 @@ function M365_BUILDER(name, value) {
             );
         }
 
-        value.domainGUID = name.replace('.', '-');
+        value.domainGUID = name.replace(/\./g, '-');
```

Comment:
Would [`replaceAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) work too? That would be more idiomatic to me.

Reply:
I agree replaceAll would be the more appropriate change, however replaceAll does not appear to be included within Otto

The Otto repo mentions it targets ES5 while replaceAll was added later in ES2021

As this regex is extremely simple, I do not believe we would see any unintended consequences or performance impact